shared/common-adapters/markdown/index.desktop.tsx

```diff
@@ -1 +1,29 @@
-export {SimpleMarkdownComponent as default} from './shared'
+import * as React from 'react'
+import {SimpleMarkdownComponent} from './shared'
+import {Props} from '.'
+
+// We convert lint heights to px here
+const DesktopSimpleMarkdownComponent = (props: Props) => {
+  const {styleOverride, ...rest} = props
+
+  const fixedStyleOverride = React.useMemo(
+    () =>
+      styleOverride
```

Comment:
There's `unifyStyles` (that `platformStyles` uses for this) in `styles/shared`, looks like you could use that